src/collections/_documentation/platforms/android/migrate.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Migration
+sidebar_order: 1
+---
+
+## Migrating from `sentry-android 1.x` to `sentry-android 2.0`
```

Comment:
the current (old sdk) sentry-android version is 1.7.x, should we use 1.7.x instead of 1.x? not sure if the same migration applies for all the 1.x versions, maybe even linking older versions to https://docs.sentry.io/clients/java/migration/ first and then the new one?

Reply:
^ @bruno-garcia and @fdrabek thoughts?

Reply:
Since I joined Sentry it's been 1.7.x
I'd say 1.x is simple and clear enough

Reply:
👍